src/components/Menu/utils.ts

```diff
@@ -12,6 +12,13 @@ export const getActiveSubMenuItem = ({ pathname, menuItem }: { pathname: string;
     return undefined
   }
 
+  // When Pathname includes "/nfts/profile", then hide subMenu.
+  if (pathname.includes('/nfts/profile')) {
```

Comment:
Good idea. But adding the `ntfs` logic here is not appropriate, the Menu component handles generic cases. Probably implement it in somewhere else ( might be in config file )

Reply:
I tried to do it in Config, but the Profile is not included in the Menu list, maybe the Profile should appear in the subNav?

e.g. Overview, Collections, Activity, Profile

Reply:
Move the Profile to the root directory, I think that solves all problems 👀